app/src/main/java/com/example/sharingang/items/ItemRepository.kt

```diff
@@ -0,0 +1,30 @@
+package com.example.sharingang.items
+
+import androidx.lifecycle.LiveData
+
+interface ItemRepository {
+    /**
+     * Returns all items
+     */
+    suspend fun getAllItems(): List<Item>
+
+    /**
+     * Add a new item
+     *
+     * The item's id must be null
+     *
+     * @return id of the item generated by the database
+     */
+    suspend fun addItem(item: Item): String?
+
+    /**
+     * Returns the item with corresponding id or null if it doesn't exist
+     */
+    suspend fun getItem(id: String): Item?
+
+    /**
+     * Returns a LiveData of all items. When the database is modified,
```

Comment:
Nitpick: probably better to not use the term "LiveData", but rather describe what you're actually returning. The fact that it's LiveData is visible from the type signature.

Also, the secondary comment should be a couple lines down, like you do with `addItem`'s comment